app/components/Calendar/Calendar.js

```diff
@@ -70,4 +72,26 @@ const Calendar = () => (
   </section>
 )
 
+const Spinner = (
```

Comment:
Is it possible to have this as a separate component? So that way future components which require async actions will be able to use this spinner?

Reply:
yeah, give me some time on it. My understanding of react is limited, but has improved some since I last worked on this.

Reply:
It looks like the deploy/netlify test failed. I'm not sure why and I'm unable to click on the details tab.